source/services/reddit.js

```diff
@@ -1,20 +1,25 @@
 /**
  * Reddit service provider
- *
- * Notes:
- * 1) How are shares counted
- * The number of upvotes minus the number of down votes for a specific URL submitted on Reddit
- * `children[0].data.score` - contain pre calculated value
+ * Share doc: https://www.reddit.com/dev/api#POST_api_submit + https://stackoverflow.com/a/32533431
+ * Counter doc: https://www.reddit.com/dev/api/#GET_search
+ * Scores are in `child.data.score` and are number of up-votes minus number of down-votes for an URL submitted to Reddit
+ * Displayed counter is calculated as sum of counts for the 5 most upvoted posts for this url.
  */
 export default {
-    counterUrl: 'https://www.reddit.com/api/info.json?url={url}&limit=1&jsonp=?',
+    counterUrl: 'https://www.reddit.com/search.json?q=url:{url}&sort=top&type=link&limit=5',
```

Comment:
It's hard to count Reddit upvotes. I've settled upon a scheme, described in the comment above.